### DISCUSSION TITLE
Enum Type or mapped selector?

### DISCUSSION THREAD

**Discussion by runnedrun** [AntlerVC/firetable#760] [Support / Q&A] [answered]
Hey folks, many of the fields are enums with numeric or string values that, in our current admin dashboard, map to pretty english names. For example:

approvalStatus:
1 -> "approved"
2 -> "rejected"
3 -> "pending"

Is there a way to make a display for this in Rowy, where "approved" is shown but the number 1 is written when that fields is selected?

Thanks!

**Comment by harinij** [comment, marked as answer]
Hi @runnedrun you can use the "Status" field type for this usecase.

You can easily configure Status field to map the field with a cell label and value like so:
<img width="275" alt="Screen Shot 2022-08-27 at 10 35 24 pm" src="https://user-images.githubusercontent.com/307298/187030524-752bea9b-7126-449e-b8bb-8b8f16826487.png">

The users of the admin panel will be able to select a field and underneath in Firestore the corresponding number value will be saved.
<img width="206" alt="Screen Shot 2022-08-27 at 10 35 54 pm" src="https://user-images.githubusercontent.com/307298/187030521-0a398952-a6ba-4b84-a514-c905198feb47.png">
